frontend/src/components/QuickAdd.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export const QuickAdd = () => {
+  return (
+    <div className="row">
+      <h2>Quick add:</h2>
+      <input
+        className="col-2 issue-label"
+        type="number"
+        min={0}
+        onChange={(event: any) => {
+          console.log(event);
+        }}
+        placeholder="Type issue number..."
+      />
+
+      <select className="col-3" name="activity" id="activity-select">
```

Comment:
The select also needs a label, for example: 
```
      <label htmlFor="activity-select" style={{ display: "none" }}>
        Choose activity
      </label>
```
(I used the id you already gave to the select).